src/test/resources/dev/sigstore/samples/rekor-response/valid/response.json

```diff
@@ -0,0 +1 @@
+{"017230556ef6b19c0dcadf4eb09805e75cfbae39d5fb57e97ea4210d5c1f552b":{"body":"eyJhcGlWZXJzaW9uIjoiMC4wLjEiLCJraW5kIjoiaGFzaGVkcmVrb3JkIiwic3BlYyI6eyJkYXRhIjp7Imhhc2giOnsiYWxnb3JpdGhtIjoic2hhMjU2IiwidmFsdWUiOiIxMzA3OTkwZTZiYTVjYTE0NWViMzVlOTkxODJhOWJlYzQ2NTMxYmM1NGRkZjY1NmE2MDJjNzgwZmEwMjQwZGVlIn19LCJzaWduYXR1cmUiOnsiY29udGVudCI6Ik1FWUNJUUR6YVR6TDNzVjZ1S0cvQ2hvb1N4c1BralZvbUlzUGFubGtVNUtSTC9Ycnl3SWhBTFZoT1JHa2pscEZnNFdtSktFMk1nSDVEYjVNN3VRNVAxaEtJSkFBZ3FobCIsInB1YmxpY0tleSI6eyJjb250ZW50IjoiTFMwdExTMUNSVWRKVGlCRFJWSlVTVVpKUTBGVVJTMHRMUzB0Q2sxSlNVTkRha05EUVdFclowRjNTVUpCWjBsSFFWbEZRakYwV0hKTlFXOUhRME54UjFOTk5EbENRVTFEVFVOdmVFUlVRVXhDWjA1V1FrRk5UVUpJVW13S1l6TlJlRWRVUVZoQ1owNVdRa0Z2VFVWSVVteGpNMUZuV1RKV2VXUkhiRzFoVjA1b1pFZFZkMGhvWTA1TmFrbDNUbFJKTWsxVWEzbE9lbEV4VjJoalRncE5ha2wzVGxSSk1rMVVhekJPZWxFeFYycEJjVTFSTUhkRGQxbEVWbEZSUkVSQlVqQmFXRTR3VFZKcmQwWjNXVVJXVVZGTFJFSkNNRnBZVGpCSlIwNXNDbU51VW5CYWJXeHFXVmhTYkUxR2EzZEZkMWxJUzI5YVNYcHFNRU5CVVZsSlMyOWFTWHBxTUVSQlVXTkVVV2RCUlU5ek9XRkxUSHA1YVZOc2JHSndXbWtLYVRGd1ZYTjJkVlpEVUZRck5WQXdTMWxoYWxGUGJsRmlkVTlKTUZKVWF6WkdibWxYYlU0d2VWWnFjVXBsS3poRWFVNWxTVGhFYjI1eFVqZFVSblphUXdwcVl6WjJiR0ZQUW5kRVEwSjJWRUZrUW1kT1ZraFJORVZHWjFGVlRGbDZiVkZTTDBoVVQxQjVkVUpITjNsUU5UUkZUbVZwVmtGTmQwaDNXVVJXVWpCcUNrSkNaM2RHYjBGVmNtRkJUM0p5TVVodlJqWkVjR1JaV1ZKT1dHSXdjbXBPUkZOVmQwUm5XVVJXVWpCUVFWRklMMEpCVVVSQloyVkJUVUpOUjBFeFZXUUtTbEZSVFUxQmIwZERRM05IUVZGVlJrSjNUVVJOUVhkSFFURlZaRVYzUlVJdmQxRkRUVUZCZDBkM1dVUldVakJTUVZGSUwwSkNSWGRFTkVWT1pFZFdlZ3BrUlVJd1dsaE9NRXh0VG5aaVZFRnlRbWR2Y2tKblJVVkJXVTh2VFVGRlFrSkNNVzlrU0ZKM1kzcHZka3d5V21oaE1sWm9XVEpPZG1SWE5UQmplVFV3Q2xwWVRqQk1iVTUyWWxSQlMwSm5aM0ZvYTJwUFVGRlJSRUZuVGtwQlJFSkhRV2xGUVRWQ1FuQkVNakpyUVRORVFWSkhSbXRuTkhseVpDdFVRWGRyYURBS1dreExlbTAwU2pnNVFTOVplblE0UTBsUlJIUmllaXQyYlZaclFuWXdSMjFSVVZCdU4ydFRWVUV4WWxOelUwTm9lSHBoY1ZsNEsxVnFhVWdyTTBFOVBRb3RMUzB0TFVWT1JDQkRSVkpVU1VaSlEwRlVSUzB0TFMwdENnPT0ifX19fQ==","integratedTime":1653593266,"logID":"d32f30a3c32d639c2b762205a21c7bb07788e68283a4ae6f42118723a1bea496","logIndex":79,"verification":{"signedEntryTimestamp":"MEUCIDX6jB8INgMJ4AkRj5T3TlvavOpwsVeex1oR5t3S37QvAiEArRn9XQ29MPwbH3MTsTdtLsY3VgdIbkXuF0Qgw4FXRao="}}}
```

Comment:
did you create a mock response and key or is this straight from rekor?

Reply:
it's straight from it's a response from rekor I just saved.